template/$PROJECT_NAME$/test/support/factory.ex

```diff
@@ -0,0 +1,11 @@
+defmodule <%= @project_name_camel_case %>.Factory do
```

Comment:
I find this a bit odd, personally. I think I'd rather have a general UserFactory like you would with FactoryGirl. That being said, no strong opinions.